### PR TITLE
Export ConnectUnreachableError

### PR DESCRIPTION
**Pull request by Fuzznimp**
This PR aims at exporting `ConnectUnreachableError`, to fine tune the error handling.

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -248,3 +248,17 @@ const key = {
 const { e, n } = key;
 const publicKey = rsaPublicKeyToPEM(n, e);
 ```
+
+## Exceptions
+
+Here is an overview of the exceptions raised from the libraries:
```
[the remark this answers is not in the document]
Comment (Fuzznimp) [reply client/README.md:254]: 🦅

**Comment by Oluane** [comment]
So, will we use this one everywhere ? 🤔

**Comment by Fuzznimp** [comment]
`ConnectUnreachableError`?

**Comment by Oluane** [comment]
Yep. In fact, what's the motivation behind this PR ?

**Comment by Fuzznimp** [comment]
Well, IMO, we should treat OAuth2 and OpenID as whole, as Connect. All the client has to know, is that Connect (as a service) is unreachable.

The motivation behind this PR is to fine tune the error handling when dealing with this lib in `Connect.Account`, and more precisely, the authorisation middleware.

**Review by Oluane — APPROVED** [review]

**Review by Oluane — DISMISSED** [review]

**Review by Oluane — APPROVED** [review]

**Pull request merged**